utils/2fa.js

```diff
@@ -0,0 +1,55 @@
+
+const prompts = require('prompts');
+const {
+    multisig: {
+        AccountMultisig: AccountMultisigApi,
+    },
+} = require('near-api-js');
+
+class AccountMultisig extends AccountMultisigApi {
```

Comment:
Seems like right now it is specific to 2FA, so makes sense to have `Account2FA`.